cypress_shared/pages/apply/applyPage.ts

```diff
@@ -52,4 +52,13 @@ export default class ApplyPage extends Page {
   checkForBackButton(path: string) {
     cy.get('.govuk-back-link').should('have.attr', 'href').and('include', path)
   }
+
+  completeOasysImportQuestions(section, sectionName: string): void {
```

Comment:
I'm a bit confused about what a section is here, partly I think because of the absence of a type. If we can't use a type could we make it a bit clearer what it is with our naming? E.g. it needs to be an array, but is singular...?

Reply:
Yep, changed to:
```
  completeOasysImportQuestions(questions: Array<OASysQuestion>, sectionName: string): void {
    questions.forEach(question => {
```